docs/endpoints/constructors.md

```diff
@@ -0,0 +1,178 @@
+[← Documentation Home](/docs/README.md)
+# Constructors
+
+Gets list of constructors 
+
+**URL** : `/ergast/f1/constructors`
+
+[Available Query Parameters](./README.md#query-parameters)
+
+---
+
+## Route Parameters:
+
+### Season
+
+**Filters only constructors that participated in a specified season. Year numbers are valid as is 'current' to get the current season list of constructors**
+
+`/{season}/` -> ex: `/ergast/f1/2024/constructors`
+
+**Note**: To utilize the `season` parameter it needs to be the first route after `/ergast/f1/`
+
+---
```

Comment:
Not just this endpoint, but the Round parameter is also available for all endpoints.

e.g. `/ergast/f1/2024/5/constructors/`, `/ergast/f1/2024/5/driverstandings/`, etc